engine/execution/computation/manager_benchmark_test.go

```diff
@@ -146,8 +146,8 @@ func BenchmarkComputeBlock(b *testing.B) {
 		Header:  &flow.Header{},
 		Payload: &flow.Payload{},
 	}
-	for _, cols := range []int{1, 4, 16} {
-		for _, txes := range []int{16, 32, 64, 128} {
+	for _, cols := range []int{16} {
+		for _, txes := range []int{128} {
```

Comment:
What was the reason for having smaller benchmarks in the first place? Was there some scaling issue we were trying to see?

Does it still make sense to keep this as loops?

Reply:
it was useful for estimating per-collection/per-block overhead.  Until we run these tests on a dedicated hardware it produces more noise in FVM benchmark report than it actually worth.